hedera-mirror-rosetta/app/domain/types/constants.go

```diff
@@ -23,6 +23,7 @@ package types
 import "github.com/coinbase/rosetta-sdk-go/types"
 
 const (
+	GeneralStatus   = "GENERAL_ERROR"
```

Comment:
```suggestion
	generalErrorStatus   = "GENERAL_ERROR"
```

symbols starting with uppercase letter is public in go, we don't want to make `generalStatus` public

also change `TransactionResults` to private,  i.e., `transactionResults`. then add the function `GetTransactionResult` here instead in `persistence/transaction.go`